inc/languages/english/admin/config_attachment_types.lang.php

```diff
@@ -6,7 +6,7 @@
  */
 
 $l['attachment_types'] = "Attachment Types";
-$l['attachment_types_desc'] = "Here you can create and manage attachment types which define which types of files users can attach to posts.";
+$l['attachment_types_desc'] = "Here you can create and manage attachment types which define which types of files users can attach to posts. Note: Disabling an attachment type will lead to removing all attachment of this type from all posts!";
```

Comment:
That verbiage is a accurate, but could be a little more concise. I'd suggest:

```suggestion
$l['attachment_types_desc'] = "Here you can create and manage attachment types which define which types of files users can attach to posts. Note: Disabling an attachment type will remove all existing attachments of this type from the forum!";
```